src/overlays/actors/ovl_Demo_Effect/z_demo_effect.c

```diff
@@ -8,7 +8,91 @@ void DemoEffect_Init(Actor* thisx, GlobalContext* globalCtx);
+void DemoEffect_UpdateGodLgtDin(DemoEffect* this, GlobalContext* globalCtx);
+void DemoEffect_UpdateGodLgtNayru(DemoEffect* this, GlobalContext* globalCtx);
+void DemoEffect_UpdateGodLgtFarore(DemoEffect* this, GlobalContext* globalCtx);
+void DemoEffect_UpdateLightRingExpanding(DemoEffect* this, GlobalContext* globalCtx);
+void DemoEffect_UpdateTriforceSpot(DemoEffect* this, GlobalContext* globalCtx);
+void DemoEffect_UpdateGetItem(DemoEffect* this, GlobalContext* globalCtx);
+void DemoEffect_UpdateLightRingShrinking(DemoEffect* this, GlobalContext* globalCtx);
+void DemoEffect_UpdateLightRingTriforce(DemoEffect* this, GlobalContext* globalCtx);
+void DemoEffect_UpdateLightEffect(DemoEffect* this, GlobalContext* globalCtx);
+void DemoEffect_UpdateJewelChild(DemoEffect* this, GlobalContext* globalCtx);
+void DemoEffect_UpdateJewelAdult(DemoEffect* this, GlobalContext* globalCtx);
+void DemoEffect_UpdateDust(DemoEffect* this, GlobalContext* globalCtx);
+void DemoEffect_UpdateCreationFireball(DemoEffect* this, GlobalContext* globalCtx);
+void DemoEffect_UpdateTimeWarpReturnFromChamberOfSages(DemoEffect* this, GlobalContext* globalCtx);
+void DemoEffect_UpdateTimeWarpPullMasterSword(DemoEffect* this, GlobalContext* globalCtx);
+void DemoEffect_UpdateTimeWarpTimeblock(DemoEffect* this, GlobalContext* globalCtx);
+
+void DemoEffect_SetupUpdate(DemoEffect* this, DemoEffectFunc updateFunc);
+void DemoEffect_JewelSparkle(DemoEffect* this, GlobalContext* globalCtx, s32 spawnerCount);
+void DemoEffect_MedalSparkle(DemoEffect* this, GlobalContext* globalCtx, s32 isSmallSpawner);
+void DemoEffect_PlayJewelSfx(DemoEffect* this, GlobalContext* globalCtx);
+void DemoEffect_SetJewelColor(DemoEffect* this, f32 alpha);
+void DemoEffect_MoveJewelSplit(PosRot* posRot, DemoEffect* this);
+void DemoEffect_MoveJewelActivateDoorOfTime(DemoEffect* this, GlobalContext* globalCtx);
+void DemoEffect_MoveJewelSpherical(f32 degrees, f32 frameDivisor, Vec3f startPos, Vec3f endPos, f32 radius,
+                                   Vec3s rotation, DemoEffect* this);
+void DemoEffect_TimewarpShrink(f32 size);
+s32 DemoEffect_DrawTimewarpLimbs(GlobalContext* globalCtx, SkelAnimeCurve* skelCuve, s32 limbIndex, Actor* thisx);
+s32 DemoEffect_CheckCsAction(DemoEffect* this, GlobalContext* globalCtx, s32 csActionCompareId);
+f32 DemoEffect_InterpolateCsFrames(GlobalContext* globalCtx, s32 csActionId);
+void DemoEffect_InitPositionFromCsAction(DemoEffect* this, GlobalContext* globalCtx, s32 csActionIndex);
+void DemoEffect_FaceToCsEndpoint(DemoEffect* this, Vec3f startPos, Vec3f endPos);
+void DemoEffect_MoveToCsEndpoint(DemoEffect* this, GlobalContext* globalCtx, s32 csActionId, s32 shouldUpdateFacing);
+void DemoEffect_MoveTowardTarget(Vec3f targetPos, DemoEffect* this, f32 speed);
+void DemoEffect_MoveGetItem(DemoEffect* this, GlobalContext* globalCtx, s32 csActionId, f32 speed);
+
+// gameplay_keep
+extern Gfx lightBall[];
```

Comment:
Are we not leaving these as D_ symbols?

Reply:
Ah I wasn't aware that was a thing in OOT. Thank you!

I'll let Fig and/or Roman chime in here. I don't see a reason to remove good names

Reply:
for things from objects specifically we havent really been naming them. feel like thats better tackled when the objects are decompiled and we arent using undefined syms anymore

Reply:
My opinion would also be to wait until we have decided on a naming scheme, like we've been doing in every other file so far.